stable/nginx-ingress/templates/controller-daemonset.yaml

```diff
@@ -29,6 +29,7 @@ spec:
 {{ toYaml .Values.controller.podLabels | indent 8}}
         {{- end }}
     spec:
+      dnsPolicy: {{ .Values.controller.dnsPolicy | default "ClusterFirst" }}
```

Comment:
Please add the default to `values.yaml` instead of using the `default` function.